django/db/models/expressions.py

```diff
@@ -1025,7 +1025,13 @@ def __init__(self, queryset, output_field=None, **extra):
 
     def __getstate__(self):
         state = super().__getstate__()
-        state.pop('_constructor_args', None)
+        args, kwargs = state['_constructor_args']
+        if args:
```

Comment:
Will that work if `queryset` is specified as a kwarg? e.g. `Subquery(queryset=queryset)`.

Reply:
Yes, I added test for this case.